platform/android/MapboxGLAndroidSDKTestApp/src/main/java/com/mapbox/mapboxsdk/testapp/activity/style/ZoomFunctionSymbolLayerActivity.java

```diff
@@ -149,6 +162,9 @@ public boolean onOptionsItemSelected(MenuItem item) {
       isInitialPosition = !isInitialPosition;
       updateSource();
     }
+    if (mapboxMap != null && item.getItemId() == R.id.menu_action_toggle_source) {
```

Comment:
We're checking `mapbox != null` twice. Here ☝️ and inside `toggleSymbolLayerVisibility()`
https://github.com/mapbox/mapbox-gl-native/blob/5882ec2ed6efee1293bb220cd7b1c7f7ea81b1f7/platform/android/MapboxGLAndroidSDKTestApp/src/main/java/com/mapbox/mapboxsdk/testapp/activity/style/ZoomFunctionSymbolLayerActivity.java#L89